src/XPurchase/VirtoCommerce.XPurchase/Schemas/WishlistResultType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GraphQL.Types;
+using VirtoCommerce.ExperienceApiModule.Core.Schemas;
+
+namespace VirtoCommerce.XPurchase.Schemas
+{
+    public class WishlistResultType : ExtendableGraphType<BulkCartResult>
+    {
+        public WishlistResultType()
+        {
+            ExtendableField<CartType>("list", "Wishlst", resolve: context => context.Source.Cart);
+            Field<ListGraphType<ValidationErrorType>>("errors", "Product errors", resolve: context => context.Source.Errors);
```

Comment:
There is no validation for `addWishlistItem`. Does it missed in `addWishlistItem` or `errors` here is redundant?